source/uwp/Visualizer/Helpers/PayloadValidator.cs

```diff
@@ -68,7 +68,8 @@ public static async Task<List<ErrorViewModel>> ValidateAsync(string payload)
                         // We don't report this error to user, it's a coding error
                         if (Debugger.IsAttached)
                         {
-                            Debugger.Break();
+                            // Commenting this out right now because its annoying, remember to un-comment before merging
```

Comment:
Do you need this change? It's better to keep it as it is

Reply:
No I don't - I just have it there while testing, will revert before merging